packages/p2p-v2/src/pages/my-ads/components/ProgressIndicator/ProgressIndicator.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { CSSProperties } from 'react';
+import clsx from 'clsx';
+import './ProgressIndicator.scss';
+
+type TProgressIndicator = {
+    className?: string;
+    style?: CSSProperties;
+    total: number;
+    value: number;
+};
+
+const ProgressIndicator = ({ className, style, total, value }: TProgressIndicator) => {
+    return (
+        <div
+            className={clsx('p2p-v2-progress-indicator__container', className)}
+            data-testid='dt_p2p_v2_progress_indicator'
+            style={style}
+        >
+            <div className={clsx('p2p-v2-progress-indicator__bar')} style={{ width: `${(value / total) * 100}%` }} />
```

Comment:
we dont need to wrap this with clsx right? or is there something missing?

```suggestion
            <div className='p2p-v2-progress-indicator__bar style={{ width: `${(value / total) * 100}%` }} />
```